sdk/src/main/kotlin/io/provenance/scope/sdk/ExecutionResult.kt

```diff
@@ -60,7 +59,7 @@ class SignedResult(session: Session, val envelope: Envelope, private val mainNet
             )
         }
 
-        if (session.proposedSession == null) {
+        if (session.scope?.sessionsList?.find { it.sessionIdInfo.sessionId.toByteArray().toUuid() == session.sessionUuid } == null) {
```

Comment:
I'm going to need to test a case for this, because I think the sessionId ByteString is in the metadata address format?

Reply:
I believe there is a `sessionUuid` inside that `sessionIdInfo` proto that you can use instead

Reply:
You're right. I moved to that in the `examples-1` branch and ran some updates successfully with that. I'll just merge this and let the code there correct it.